app/models/activity.rb

```diff
@@ -334,9 +334,9 @@ def own_and_descendants_associates(model)
     model.where(parent_activity_id: activity_ids)
   end
 
-  def reportable_actuals_for_level
+  def reportable_transactions_for_level
     if programme?
-      spend_by_financial_quarter(own_and_descendants_associates(Actual).order("date DESC"))
+      spend_by_financial_quarter(own_and_descendants_associates(Transaction).order("date DESC"))
     else
       actuals.order("date DESC")
```

Comment:
Is this what you meant that at other levels, it's summing up the actuals but the lack of adjustments/refunds hasn't been flagged? If so, do you think it's worth asking Dan in Slack to double-check?